src/components/contractCode.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="contract-code">
+    <ae-switch
+      v-model="view"
+      name="example"
+      :choices="[
+        { label: 'Hex', value: 'hex' },
+        { label: 'Opcode', value: 'opcode' },
+      ]"
+      default="hex"
+    />
+    <code-view :code="code" />
+  </div>
+</template>
+
+<script>
+import { AeSwitch } from '@aeternity/aepp-components'
+import CodeView from './codeView.vue'
+import numbersToString from '../filters/numbersToString'
+import aevmDisassembler from '../filters/aevmDisassembler/index'
+
+export default {
+  name: 'contract-code',
+  props: ['contractCode'],
+  components: {
+    AeSwitch,
+    CodeView
+  },
+  data () {
+    return {
+      view: null
+    }
+  },
+  computed: {
+    code () {
+      switch (this.view) {
+        case 'opcode': return aevmDisassembler(this.contractCode)
+        default: return numbersToString(this.contractCode)
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../style/variables';
+
+.contract-code {
+  .ae-switch {
+    padding-right: 20px;
+    justify-content: flex-end !important;
+    background: lighten($black, 10);
+
+    /deep/ label {
```

Comment:
Usually, Vue components shouldn't modify styles of child components. `scoped` mode prevents unintentional changing of styles of other components. In case if we still want to change styles of nested components in `scoped` mode we should use `/deep/` combinator.
https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors